experimental/devices/cap1188/cap1188_test.go

```diff
@@ -277,24 +263,24 @@ func TestDev_InputStatus(t *testing.T) {
 		bus := &i2ctest.Playback{
 			Ops: append(setupPlaybackIO(), []i2ctest.IO{
 				{Addr: 40, W: []byte{0x3}, R: []byte{1 << 7}},
-				{Addr: 40, W: []byte{0x10}, R: []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}},
-				{Addr: 40, W: []byte{0x30}, R: []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}},
+				//{Addr: 40, W: []byte{0x10}, R: []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}},
```

Comment:
I'm surprised you were able to remove those

Reply:
It works because it is in the same package. That's the idiomatic way.